melpazoid/melpazoid.el

```diff
@@ -399,9 +399,10 @@ CASE-INSENSITIVE determines the case-sensitivity of the matches."
 
 (defun melpazoid--annotate-line (msg)
   "Annotate the current line with MSG."
-  (melpazoid-insert "- %s#L%s: %s"
+  (melpazoid-insert "- %s:%s:%s: %s"
                     (file-name-nondirectory (buffer-file-name))
                     (line-number-at-pos)
+                    (1+ (- (point) (line-beginning-position)))
```

Comment:
Thank you!  I'm definitely for this change, but would this achieve the same?
```suggestion
                    (current-column)
```
Also I notice that this will return 1-indexed column numbers instead of 0-indexed.  In elisp if you do e.g. `(move-to-column 1)` it will put you 1 column from the left margin, so I think 0-indexed should be the preference?